_includes/find-help/main-title.html

```diff
@@ -1,41 +1,31 @@
-<div class="shadow-sm p-3 mb-2 bg-white rounded">
-  <div class="row">
-    <div class="col-1">
-      <div class="float-left">
-        <button
-          class="navbar-toggler collapsed hamburger-background light-purple-background align-left"
-          type="button"
-          data-toggle="collapse"
-          data-target="#navbarsExampleDefault"
-          aria-controls="navbarsExampleDefault"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-          onclick="on()"
-        >
-          <span class="icon-bar top-bar color-black"></span>
-          <span class="icon-bar middle-bar color-black"></span>
-          <span class="icon-bar bottom-bar color-black"></span>
-        </button>
-      </div>
+<div class="jumbotron mb-1">
```

Comment:
since you're using the jumbotron now, I wonder if we can make the navbar template somehow reusable with an include for custom content?
That is, can you unify the header/navbar for the main page with the one for the find-help page, by having a parameter for the content that goes between the hamburger icon and the "cantstopcolumbus" logo?

Reply:
That's not a bad idea.

Not sure how but I'm sure it is doable.

Reply:
do you want to get after it, or should I approve this PR and you'll investigate at a later date?

Reply:
Later

Reply:
Later